openpype/pipeline/publish/lib.py

```diff
@@ -757,6 +698,7 @@ def get_publish_repre_path(instance, repre, only_published=False):
         str: Path to representation file.
         None: Path is not filled or does not exists.
     """
+    from .. import get_instance_staging_dir
```

Comment:
Aren't we avoiding relative imports almost everywhere else?

Reply:
Is it problem?? Or is there a better way to do it?

Reply:
ok, so I have figured out with the assistance of @iLLiCiTiT ;)